Index: b/buildbot/steps/package/deb/pbuilder.py
===================================================================
--- a/buildbot/steps/package/deb/pbuilder.py
+++ b/buildbot/steps/package/deb/pbuilder.py
@@ -44,7 +44,7 @@ class DebPbuilder(WarningCountingShellCo
     architecture = None
     distribution = 'stable'
     basetgz = "/var/cache/pbuilder/%(distribution)s-%(architecture)s-buildbot.tgz"
-    mirror = "http://cdn.debian.net/debian/"
+    mirror = "http://httpredir.debian.org/debian/"
     extrapackages = []
     keyring = None
     components = None
Index: b/buildbot/test/unit/test_steps_package_deb_pbuilder.py
===================================================================
--- a/buildbot/test/unit/test_steps_package_deb_pbuilder.py
+++ b/buildbot/test/unit/test_steps_package_deb_pbuilder.py
@@ -43,7 +43,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/stable-local-buildbot.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/'])
+                                 '--mirror', 'http://httpredir.debian.org/debian/'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
                         command=['pdebuild', '--buildresult', '.',
@@ -94,7 +94,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/stable-amd64-buildbot.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/',
+                                 '--mirror', 'http://httpredir.debian.org/debian/',
                                  '--architecture', 'amd64'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
@@ -115,7 +115,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/woody-local-buildbot.tgz',
                                  '--distribution', 'woody',
-                                 '--mirror', 'http://cdn.debian.net/debian/'])
+                                 '--mirror', 'http://httpredir.debian.org/debian/'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
                         command=['pdebuild', '--buildresult', '.',
@@ -134,7 +134,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/buildbot/stable-local.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/'])
+                                 '--mirror', 'http://httpredir.debian.org/debian/'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
                         command=['pdebuild', '--buildresult', '.',
@@ -172,7 +172,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/stable-local-buildbot.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/',
+                                 '--mirror', 'http://httpredir.debian.org/debian/',
                                  '--extrapackages', 'buildbot'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
@@ -193,7 +193,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/stable-local-buildbot.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/',
+                                 '--mirror', 'http://httpredir.debian.org/debian/',
                                  '--debootstrapopts', '--keyring=/builbot/buildbot.gpg'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
@@ -213,7 +213,7 @@ class TestDebPbuilder(steps.BuildStepMix
                         command=['sudo', '/usr/sbin/pbuilder', '--create',
                                  '--basetgz', '/var/cache/pbuilder/stable-local-buildbot.tgz',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/',
+                                 '--mirror', 'http://httpredir.debian.org/debian/',
                                  '--components', 'main universe'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
@@ -242,7 +242,7 @@ class TestDebCowbuilder(steps.BuildStepM
                         command=['sudo', '/usr/sbin/cowbuilder', '--create',
                                  '--basepath', '/var/cache/pbuilder/stable-local-buildbot.cow/',
                                  '--distribution', 'stable',
-                                 '--mirror', 'http://cdn.debian.net/debian/'])
+                                 '--mirror', 'http://httpredir.debian.org/debian/'])
             + 0,
             ExpectShell(workdir='wkdir', usePTY='slave-config',
                         command=['pdebuild', '--buildresult', '.',
